### ISSUE TITLE
Load multiple local files

### ISSUE THREAD

**Issue by vredesbyyrd** [skorokithakis/catt#337]
Hi, thanks for sharing `catt`, its great.  

Would it be possible to implement some sort of "queue" for local media files, eg, `catt cast /home/user/some-directory` would load every file in the directory. Perhaps `catt cast file1 file2 ...`,  or maybe even reading filepaths from a text file? Some method to load more than one local file would be very nice to have.  

Thanks for your time.

**Comment by theychx** [comment]
Thats a reasonable request, and certainly doable. @skorokithakis ?

**Comment by skorokithakis** [comment]
Yeah, this does sound good. IIRC we already have the functionality to wait for the current file to finish, so this shouldn't be too hard, is that accurate?

**Comment by theychx** [comment]
@skorokithakis Indeed, I'll give this a shot.

**Comment by vredesbyyrd** [comment]
@theychx  

Thanks for giving this a go. One more thought regarding this. If you were to implement it with my first example of playing all files in dir, `catt cast /home/user/some-directory`, having a flag to optionally define the file to start from would be nice.  eg `catt cast --index=3 /home/user/some-directory`, where `--index=3` represents the third (or 4th if counting from 0) file in directory.  

I know this may complicate things....Just an idea!

**Comment by skorokithakis** [comment]
That seems a bit overcomplicated, I think the simplest way is to either specify all the files in the command line one by one or read them from a file, one per line.

Just doing the former is probably okay for now, I'd think.

**Comment by vredesbyyrd** [comment]
Right on. As a user I would be happy with either of the methods you described. And on second thought I would agree, index flag probably over complicates this when we can define wanted files on command line or text file.

**Comment by chapmanjacobd** [comment]
It's possible to do this by calling catt multiple times. 

For example in `fish` shell:

```fish
for file in (fd . ~/d/80_Now_Listening/ -tf | shuf -n50)
    catt cast "$file"
end
```

In Windows cmd:

```
FOR %%I in (C:\User\me\Music\*.mp3) DO catt cast %%I 
```

If you wanted to skip the first 4 files you can also do that:

```fish
for file in (fd . ~/d/80_Now_Listening/ -eMP3 | tail -n +4)
    catt cast "$file"
end
```

**Comment by chapmanjacobd** [comment]
I made something that can keep track of what songs you played as well as YouTube subscriptions, and many other things:

https://github.com/chapmanjacobd/library

It uses catt under the hood and has full-text search capabilities

**Comment by fuglede** [comment]
This would be great for making slideshows of photo albums! Looping over all files sort of works but causes black screens in between when I try.

It would be nice if the end result also supported the "previous track"/"next track" buttons that show up in e.g. Google Home, the various widgets etc..
